src/molecules/downtime-page/index.tsx

```diff
@@ -4,6 +4,7 @@ import { Avatar, Box, Button, Typography } from "@mui/material";
 import CallRoundedIcon from "@mui/icons-material/Call";
 import { useColorPalates } from "../../molecules/theme-provider/hooks";
 import { useUiConfig } from "../../hook/useConfig";
+import downtimeImage from ".././downtime-page/assets/downTimeGIF.gif";
```

Comment:
Hey @soumya-maheshwari we wanted the image to come from config to maintain linearity across all the other molecules. Kindly import the image from Config only

Reply:
ok got it

Reply:
> Hey @soumya-maheshwari we wanted the image to come from config to maintain linearity across all the other molecules. Kindly import the image from Config only

i just wanted to confirm that the  deployment environment is configured to serve static files, including GIFs. 
because GIF is rendering on localhost but on deployment.